src/Components/Aspire.Azure.AI.OpenAI/AspireAzureOpenAIExtensions.cs

```diff
@@ -90,7 +90,7 @@ protected override IAzureClientBuilder<AzureOpenAIClient, AzureOpenAIClientOptio
 
                     if (!string.IsNullOrEmpty(settings.Key))
                     {
-                        var credential = new AzureKeyCredential(settings.Key);
+                        var credential = new ApiKeyCredential(settings.Key);
```

Comment:
This is working around a binary breaking change between beta versions of System.ClientModel 1.1.0. See https://github.com/Azure/azure-sdk-for-net/pull/45554. The Azure.AI.OpenAI 2.0.0-beta5 version we are using was using that implicit operator, which is no longer there in the released 1.1.0 GA version. Using the ApiKeyCredential directly side-steps the implicit operator break.